src/app/utils/react-query-hooks.ts

```diff
@@ -25,7 +25,7 @@ export const useItems = (initialData: Array<Table<'items'>>) => {
 
```

Comment:
I think these helper files can be deleted, they were inherited. Sorry, looks like you've had to mess around loads with types here!

Reply:
ah fair! will do